core/src/main/java/tech/pegasys/web3signer/core/multikey/metadata/BlsArtifactSignerFactory.java

```diff
@@ -99,6 +101,15 @@ public ArtifactSigner create(final InterlockSigningMetadata interlockSigningMeta
     }
   }
 
+  @Override
+  public ArtifactSigner create(final YubiHsmSigningMetadata yubiHsmSigningMetadata) {
+    try (TimingContext ignored = privateKeyRetrievalTimer.labels("interlock").startTimer()) {
```

Comment:
The metric label should be yubihsm not interlock

Reply:
👍  ... fixed.